app/src/main/java/com/navapp/navigation/util/geofencing/UponArrival.java

```diff
@@ -0,0 +1,12 @@
+package com.navapp.navigation.util.geofencing;
```

Comment:
How is this `util`? It's a component of the application

Reply:
IDK man